polynote-server/src/main/scala/polynote/server/SharedNotebook.scala

```diff
@@ -324,8 +328,50 @@ class IOSharedNotebook(
 
     override def info: IO[Option[KernelInfo]] = ifKernelStarted(_.info, None)
 
-    override def getHandleData(handleType: HandleType, handle: Int, count: Int): IO[List[ByteVector32]] =
-      ensureKernel().flatMap(_.getHandleData(handleType, handle, count))
+    private val streams = new mutable.HashMap[Int, Iterator[ByteVector32]]
```

Comment:
I think something like `streamHandles` might be a better name, since there are all sorts of streams hangin' out round here?

Reply:
👍